src/affinder/napari.yaml

```diff
@@ -0,0 +1,10 @@
+name: affinder
+schema_version: 0.1.0
+contributions:
+  commands:
+  - id: affinder.start_affinder
+    title: Create start_affinder
+    python_name: affinder:start_affinder
+  widgets:
+  - command: affinder.start_affinder
+    display_name: Start affinder
```

Comment:
We also need to remove the `@napari_hook_implementation` decorator from npe1 in the `_plugin` file?

Reply:
I deleted the `plugin` file and moved the `start_affinder` import to the `init`

Reply:
Perfect! I love it! :smiley: